### DISCUSSION TITLE
Svelecte filtering by first letters in string

### DISCUSSION THREAD

**Discussion by stonesha** [mskocik/svelecte#175] [Q&A] [answered]
Would there be any way to match on the first letters of a string? For example in the picture below I would only want "Memphis" to show up and not match the "me" in "Sacramento". For clarification I would like to be able to do this client-side and not have to filter server-side and fetch the results. Thanks!

![image](https://github.com/mskocik/svelecte/assets/36431006/cfb0cb04-27ef-4e54-aa91-0687388c2bd4)

Here's a minimal reproduction if you need it.
https://svelte.dev/repl/7f4e3a25837b4a3ab26979742fb00a8a?version=3.59.1

 Thanks for your library as it's incredibly useful and I'm using it a lot!

**Comment by mskocik** [comment]
This is not possible at the moment, but should be quite easy to add 👍

**Reply by mskocik** [reply, marked as answer]
This took a while but it has been added in`4.1.`

**Reply by stonesha** [reply]
Thank you so much!
